thorlcr/graph/thgraph.hpp

```diff
@@ -1186,6 +1186,7 @@ class graph_decl CActivityBase : implements CInterfaceOf<IThorRowInterfaces>, im
     unsigned getOptUInt(const char *prop, unsigned defVal=0) const { return container.getOptUInt(prop, defVal); }
     __int64 getOptInt64(const char *prop, __int64 defVal=0) const { return container.getOptInt64(prop, defVal); }
     unsigned __int64 getOptUInt64(const char *prop, unsigned __int64 defVal=0) const { return container.getOptUInt64(prop, defVal); }
+    virtual stat_type getNodeSpillSize(unsigned node) const { return 0; }
```

Comment:
minor: as this is always going to be a manager, not worker method, it should probably be in CMasterActivity only (and cast in master code when accessing it).

Reply:
Can we clarify the var/method names, by renaming xxxNodeSpillxxx to xxxActivitySpillxxx, and xxxNodeGraphxxx to xxxInterGraphSpillxxx ?